docs/components/forms_advanced.rst

```diff
@@ -0,0 +1,188 @@
+Forms - advanced
+################
+
+Sometimes you might have more complex requirements for your Forms. You might already be aware that Mautic builds upon the Symfony framework, so there's lots of Form-related features available to you.
+You can read more about this here: :xref:`Symfony 4 form classes`. This section highlights a few complex but common use cases:
+
+- Custom Form Types that deviate from Symfony's wide array of standard Form types, like text fields, choice fields, etc.
+- Data sanitizing
+- Dynamic Form modification
+- Data validation
+
+Custom Form types
+*****************
```

Comment:
Vale reports the following (`Mautic.FeatureList`):

<img width="564" alt="image" src="https://user-images.githubusercontent.com/17739158/204145231-a9c3ad04-6955-4000-8672-d18ce24727b3.png">

... but when I change it to `Custom Form types`, it triggers the following warning (`Google.Headings`):

<img width="549" alt="image" src="https://user-images.githubusercontent.com/17739158/204145240-0beb30b0-05d9-4440-95d2-4e54dca01cc4.png">

Any suggestions @RCheesley? We could disable `Google.Headings` in the entire project if you want, or we can put a `.. vale off` and `.. vale on` around this heading. I'd be in favor of the first one as I can already see lots of new contributors running into this. WDYT?

Reply:
Just use the following:

```
.. vale off

Heading Goes Here

.. vale on
```

if you need to use camel case to be in keeping with our styleguide :) We use it a lot in the end-user docs so please do use it here too :)

Reply:
Thanks! This one should be good to go now @RCheesley 😊